configure.ac

```diff
@@ -147,6 +148,10 @@ else
   if test "$CONFIG_HOST" = "Darwin" -a `basename "$AS_LD"` = "ld"; then
     AC_MSG_ERROR([Please install lld or i386-elf-binutils])
   fi
+  if test `basename "$AS_LD"` = "ld.lld"; then
```

Comment:
Can you check that against
gnu ld and just remove the
check if it works? Fewer
special-casings.

Reply:
It only works in binutils since 2.44: https://lists.gnu.org/archive/html/info-gnu/2025-02/msg00001.html
which is a bit too new still.

A portable solution would involve a linker script. Like this
https://patchwork.yoctoproject.org/project/oe-core/patch/20250908210657.2959408-3-raj.khem@gmail.com/
would you prefer that?

Reply:
Ah, never mind then.
Or enrich the comment, if you want. :)